OpenRA.Mods.Common/Activities/Move/MoveWithinRange.cs

```diff
@@ -53,5 +54,18 @@ bool AtCorrectRange(WPos origin)
 		{
 			return Target.IsInRange(origin, maxRange) && !Target.IsInRange(origin, minRange);
 		}
+
+		bool InCorrectLayer(Actor self)
```

Comment:
The Game/Mods split means we can't put this on Target itself, but it does make sense to deduplicate it somewhere.